src/Microsoft.AspNetCore.Blazor.Build/Cli/Commands/ResolveRuntimeDependenciesCommand.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Blazor.Build.Core;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace Microsoft.AspNetCore.Blazor.Build.Cli.Commands
+{
+    class ResolveRuntimeDependenciesCommand
+    {
+        public static void Command(CommandLineApplication command)
+        {
+            var references = command.Option("--reference",
+                "Full path to the application dependencies that are not part of netstandard2.0",
+                CommandOptionType.MultipleValue);
+
+            var baseClassLibrary = command.Option("--base-class-library",
+                "Full path to the directories with the dlls that make up the mono BCL",
+                CommandOptionType.MultipleValue);
+
+            var resolutionOverrides = command.Option(
```

Comment:
I can't see any indication that this is used anywhere (I mean, the value gets passed through the code, but it's never used to affect the output). Can this be removed? Or am I missing something?

Reply:
This was before our talk about us mimicking the code the linker uses to pick the final list of assemblies. The idea was to allow specifying overrides to the algorithm. I think we won't need it, so I'll take it out.